lang/English.ini

```diff
@@ -241,7 +241,7 @@ DIRECT = "Direct Connection"
 RULES_TITLE = "RULES"
 RULE_1 = "1. You must be 13 years or older."
 RULE_2 = "2. No hate speech, slurs, or any kind of offensive language."
-RULE_3 = "3. Do not use any modified build."
+RULE_3 = "3. Do not use any modified build or rom"
```

Comment:
don't need to change this. coop will reject a rom that has the wrong hash

Reply:
Ah ok 👍